packages/engine_umbrella/apps/engine/lib/engine/application.ex

```diff
@@ -54,7 +54,9 @@ defmodule Engine.Application do
       Engine.AdminProcessor,
       Engine.BookingUpdatesProcessor,
```

Comment:
Probably need to rename this to not confuse it with the new BookingUpdateProcessor
same for the vehicle one